Changelog.md

```diff
@@ -1,27 +1,142 @@
-# 1.1.26 -- 2018-07-06
+
+# Changelog
+
+## next version
+
+- Make `jsonErrors` total.
```

Comment:
In addition to merging the two changelog files, I also added an entry describing this PR's change.